### ISSUE TITLE
Better Debugging features : Ability to trace the exceptions or issues

### ISSUE THREAD

**Issue by musanas** [a85/POSTMan-Chrome-Extension#5252]
Hello,

**Is your feature request related to a problem? Please describe.**
Yes, 
I do have a collection which is relatively a larger one with many requests and many Tests
I often run them either using postman Runner fetaure, or with Newman cmd tool
I do have some testing functions written at **pre-request scripts** tab of the Collection or at a particular folder level.
I tried to make this pre-request scripts in such a way that, I write it only once at Collection or Folder level, then I re-use/invoke the same js functions at **Tests** tab of individual request endpoints present in the Collection.

I do come across some issues with the api endpoints sometimes due to various reasons that it do not return a response content or in other words give a empty body etc. And the functions that I invoke are called with these response arguments.

Now sometimes those functions will not work as expected because the passed argument could be of empty or something . (The issue could be something like, I have a empty response, and in the common function if I try to parse it to JSON format , it will give JSON exception, I could have handled it any way at the functiona definition level, but thats not the issue am trying to say here)

and in such situations the Postman will give notification that "**Something went wrong while running your scripts. Check Postman Console for more info.**"

Looking at the console will only give some idea about what is the issue
It do not give idea on where exactly the issue had happened
i.e it do not give a stack trace kind of information
So its some times bit difficult / time consuming for me to identify where exactly in the script/function the isssue is happening in order to fix them or take action. One good thing is the console at lease give idea on under what request hit the issue had happen.

**Describe the solution you'd like**
Better debuggin facility
Trace of errors in the console window

**Describe alternatives you've considered**
Putting console.log in the script, but that looks not good for me.
Eventually I do not need many console.log in my script, and i will have to clean them up once i traced down the issue

**Additional context**
Please find the screenshot of information in the current console window
![postman console](https://user-images.githubusercontent.com/25841256/46059398-afe15900-c191-11e8-96ce-f6a534695fde.png)

**Comment by nikithaachhini23** [comment]
Hi @musanas 👋 

Thanks for reporting this issue!

We’ve now introduced a new `Errors` tab for collection runs, so any script or runtime errors are now surfaced per request in the run result.

Here’s a quick screenshot of how it looks:

<img width="1093" height="509" alt="Image" src="https://github.com/user-attachments/assets/84e5aed7-c913-4104-bab6-9ab9b1d32165" />

Update to the latest version **v11.75.1** to use it, and let us know how it works for you!  

We understand the need for a full stack trace as well, and we’ll update you once support for it is added.

**Comment by jauntyjocularjay** [comment]
Adding to this because -- this isn't terribly helpful.

<img width="897" height="811" alt="Image" src="https://github.com/user-attachments/assets/1477b259-6a69-42a5-96e2-fea49226dab8" />

Given this information, where is the problem occurring? Is it in my pre request script? Post request script? Package? This function is definitely defined because I used it in other places.

Asking the LLM isn't helpful either.

<img width="614" height="615" alt="Image" src="https://github.com/user-attachments/assets/9f1fe812-e600-448f-af6c-2b5f7f4a7cdf" />

I have encapsulated my own `environment.setDevice()` that uses `pm.environment.set()` but there is no way to add my package to the context so the AI can't see what is actually running. The LLM is an entirely different set of problems.

After some debugging I finally discovered oh, this is a typo. Which means I now know the error was in the package and I broke my post-request script when I updated the function name.

**Comment by AnnaVernerovaHID** [comment]
I agree with the above posters. When scripts throw messages (as in: there is actual error in the script itself, as opposed to a failed assertion/test), they should be accompanied with a stackTrace. I often get messages such as `JSONError:"undefined" is not valid JSON` but it's absolutely not clear from which script the message is coming - is it from a pre-request script or a post-response (test) script? Folder level script? Request level script? Which line? Which variable was undefined when I did not expect it to be?
